### ISSUE TITLE
A few suggestions

### ISSUE THREAD

**Issue by ginsm** [LoremVIpsum/OpenSimpleFrames#21]
I haven't really looked through the code so maybe there's some limitations that'd make some of these suggestions impractical.. but figured I'd throw them out there anyway:

1. The frame block's preferred tool should be an axe as they break quite slowly currently.
2. The frame block should only be able to be set once and to change it you should need to break and replace the frame. Or maybe require the player to be holding a specific item in their off hand in order to set the frame's texture (maybe the specific frame block)? Currently it's super easy to accidentally reset the frames.
3. Could the block be consumed when setting the texture and returned after changing the texture or breaking the frame? It'd be a nice balance feature. :thinking:

Honestly, even if the third suggestion doesn't happen, I'd love if the first two were to happen.

**Comment by LoremVIpsum** [comment]
yeah currently they dont have a tool but probably should, I think item consumption would be difficult to track in this current version, but something like that could be good so they arent as easy to reset

**Comment by baptisteImbert** [comment]
I agree with all suggestions made by ginsm, especially the third. Having an option in a config file to chose if the block should be consumed when applying its texture would be great! In my case, even losing that block when breaking the frame would be better than the curent implementation. Thanks for the mod by the way!
